packages/cf.js/src/node/index.ts

```diff
@@ -1,5 +1,7 @@
+import { ethers } from "ethers";
```

Comment:
We should be consistently using `import ethers from "ethers"` @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll